src/docs/_layouts/default.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```

Comment:
that's a default jekyll layout used for all three files (basic/components/containers). It will be easy to add one navigation now :fireworks: